debian/changelog

```diff
@@ -1,3 +1,11 @@
+eos-watermark-extension (3.9.0rc1) eos; urgency=medium
```

Comment:
We don't normally keep per-change changelogs for first-party components. We rely on our CI infrastructure incrementing the version number. I'd just call it 3.9.0, or leave this out.

Reply:
If you really want a new changelog entry and don't want to risk  needing to force-push tags you could bump the version & add the changelog entry as the first commit in the PR, and tag that?

Reply:
Wonderful

Reply:
I would just drop this changelog entry altogether and update it to 3.9.0 in the final version.